docs/release-notes.md

```diff
@@ -1,5 +1,47 @@
 # Release notes
 
+## Trino Gateway 14 (xx Feb 2025) { id="14" }
+
+Artifacts:
+
+* [JAR file gateway-ha-14-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/14/gateway-ha-14-jar-with-dependencies.jar)
+* Container image `trinodb/trino-gateway:14`
+* Source code as
+  [tar.gz](https://github.com/trinodb/trino-gateway/archive/refs/tags/14.tar.gz)
+  or [zip](https://github.com/trinodb/trino-gateway/archive/refs/tags/14.zip)
+* [Trino Helm chart](https://trinodb.github.io/charts/) `trino/trino-gateway` version `1.14.0`
+
+Changes:
+
+* Add display of the cluster health status on the cluster page.
+  ([#601](https://github.com/trinodb/trino-gateway/pull/601))
+* Add page to show and edit routing rules.
+  ([#433](https://github.com/trinodb/trino-gateway/pull/433))
+* Add support for automatic database schema validation and upgrade during Trino Gateway startup.
+  ([#575](https://github.com/trinodb/trino-gateway/pull/575))
+* Enable use of Oracle as backend database for Trino Gateway.
+  ([#588](https://github.com/trinodb/trino-gateway/pull/588))
+* Add support to configure HTTP client options for the external routing group selector. 
+  ([#580](https://github.com/trinodb/trino-gateway/pull/580))
+* Add support for monitoring Trino Gateway with JMX.
```

Comment:
IIUC, this implements cluster health monitor using JMX for Trino cluster, not for the gateway. (similar to the OpenMetrics monitor PR)

Reply:
oops